datastore/src/test/java/io/spine/server/storage/datastore/TransactionWrapperTest.java

```diff
@@ -139,6 +141,52 @@ void readWrite() {
                 .isEqualTo(updated.getLong(field));
     }
 
+    @Test
+    @DisplayName("delete single entity and many entities in a bulk by their keys transactionally")
+    void writeDelete() {
+        int entityCount = 20;
+        List<Entity> entities =
+                generate(() -> Entity.newBuilder(keyFactory.newKey(newUuid()))
+                                     .set("uuid", newUuid())
+                                     .build())
+                        .limit(entityCount)
+                        .collect(toList());
+
+        TransactionWrapper creationTx = datastore.newTransaction();
+        creationTx.create(entities);
+        creationTx.commit();
+
+        Entity firstEntity = entities.remove(0);
+        TransactionWrapper singleDeletionTx = datastore.newTransaction();
+        Key toDelete = firstEntity.getKey();
+        singleDeletionTx.delete(toDelete);
+        singleDeletionTx.commit();
+
+        TransactionWrapper firstReadTx = datastore.newTransaction();
+        Truth8.assertThat(
```

Comment:
Please add a static import for that.

Reply:
The beauty of static imports is in the fact that you *can* import methods with the same name, as long as they don't clash parameter-wise.